### ISSUE TITLE
Update for JQuery 3.X.X

### ISSUE THREAD

**Issue by mlintern** [farbelous/fontawesome-iconpicker#32]
I get an error trying to open an input popover with the latest version of JQuery (3.1.1):

```
jquery-3.1.1.min.js:4 Uncaught TypeError: f.getClientRects is not a function
    at r.fn.init.offset (jquery-3.1.1.min.js:4)
    at Object.getWithinInfo (fontawesome-iconpicker.min.js:1)
    at r.fn.init.a.fn.pos (fontawesome-iconpicker.min.js:1)
    at c.updatePlacement (fontawesome-iconpicker.min.js:1)
    at c.show (fontawesome-iconpicker.min.js:1)
    at c.toggle (fontawesome-iconpicker.min.js:1)
    at HTMLSpanElement.<anonymous> (fontawesome-iconpicker.min.js:1)
    at HTMLSpanElement.dispatch (jquery-3.1.1.min.js:3)
    at HTMLSpanElement.q.handle (jquery-3.1.1.min.js:3)
```

**Comment by mstojanovv** [comment]
I have the same error, any solutions for this?

**Comment by HAYMbl4** [comment]
I found [question](http://stackoverflow.com/questions/37914869/jquery-ui-error-f-getclientrects-is-not-a-function) on stackoverflow. It's may be helpful ([JQuery 3 Migrate plugin](http://code.jquery.com/jquery-migrate-3.0.0.js))

**Comment by fnf1993** [comment]
I have the same error, any solutions for this?

**Comment by mstojanovv** [comment]
Hi @fnf1993 read the previous reply about the jquery migrate plugin, this solves the issue.

**Comment by itsjavi** [comment]
fixed as of v1.3.1 it was a compat. problem with jQuery UI position
